config/components/webhook/podwebhook.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
is this still valid? I guess not, right?

Reply:
deleted